src/mux-analytics.brs

```diff
@@ -376,7 +417,11 @@ function muxAnalytics() as Object
 
   prototype.videoAddedHandler = sub(video as Object)
     m._videoProperties = m._getVideoProperties(video)
-    m._videoContentProperties = m._getVideoContentProperties(video.content)
+    if video.contentIsPlaylist = true
+      m._videoContentProperties = m._getVideoContentProperties(video.content.getChild(video.contentIndex))
```

Comment:
likewise just making sure, we don't need to validate that `contentIndex` isn't invalid, do we? sorry, my roku/brightscript knowledge is iffy, and makes me be way too cautious

Reply:
Yes, we don't need to validate `contentIndex` as it is the video node's field and has a default value of `false`.